asg/ecs.tf

```diff
@@ -23,9 +24,51 @@ resource "aws_ecs_service" "webapp_service" {
     }
 }
 
+resource "aws_ecs_service" "webapp_service_fargate" {
+    count = "${var.launch_type == "FARGATE" ? 1 : 0}"
+    name = "${var.name_prefix}_webapp_service"
+    cluster = "${aws_ecs_cluster.webapp_cluster.id}"
+    task_definition = "${aws_ecs_task_definition.webapp_definition_fargate.arn}"
+    launch_type = "FARGATE"
+    desired_count = "${var.count_webapp}"
+    deployment_minimum_healthy_percent = "${var.minimum_healthy_percent_webapp}"
+    iam_role = "${var.ecs_service_role}"
```

Comment:
With Fargate, we don't need to create `ecs_service_role` or even pass `iam_role` here, since Fargate will automatically use service-linked roles for ECS.

See: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using-service-linked-roles.html

It might be good to put this URL in the Readme, in case someone needs to use custom rule service linked roles.

For now, I think it's a good idea to put `launch_type` variable in `static` and we can decide whether we need to create it via `count = "${var.launch_type == "FARGATE" ? 1 : 0}"`.